nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -61,6 +61,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 - `mkosi` was updated to v19. Parts of the user interface have changed. Consult the
   [release notes](https://github.com/systemd/mkosi/releases/tag/v19) for a list of changes.
 
+- The package `optparse-bash` is now dropped due to upstream inactivity. [Alternatives](https://github.com/shadawck/awesome-cli-frameworks#bashshell) available in Nixpkgs include [`argc`](https://github.com/sigoden/argc), [`argbash`](https://github.com/matejak/argbash), [`bashly`](https://github.com/DannyBen/bashly) and [`gum`](https://github.com/charmbracelet/gum), to name a few.
```

Comment:
Is the awesome link needed?

Reply:
Not really, especially since all the options it provides are emumerated.

Reply:
Addressed.